benchmark/operation_test.go

```diff
@@ -133,16 +146,22 @@ func BenchmarkClientRead(b *testing.B) {
 			if err != nil {
 				b.Fatalf("error deleting the document: %v", err)
 			}
-			_, err = coll.InsertOne(context.Background(), bson.D{{"text", text}})
-			if err != nil {
-				b.Fatalf("error inserting one document: %v", err)
+			for i, text := range teststrings {
+				_, err = coll.InsertOne(context.Background(), bson.D{{"_id", i}, {"text", text}})
+				if err != nil {
+					b.Fatalf("error inserting one document: %v", err)
+				}
 			}
 
 			b.ResetTimer()
 			b.RunParallel(func(p *testing.PB) {
 				for p.Next() {
+					n, err := rand.Int(rand.Reader, big.NewInt(2))
```

Comment:
Same as above: Consider incrementing a counter and using `teststrings[i % len(teststrings)]` instead of using a random number.